src/lang/po/fr/Prusa-Firmware-Buddy_fr.po

```diff
@@ -344,14 +344,14 @@ msgstr ""
 
 #: src/gui/screen_home.cpp:83 src/gui/screen_home.cpp:82
 msgid "HOME"
-msgstr "POSITION INITIALE"
+msgstr "ORIGINES XYZ"
```

Comment:
this is the title of the main screen - can you think about some undestandable name for it?

Reply:
if it's for the main screen as well put "acceuil"